Makefile

```diff
@@ -95,16 +95,17 @@ $(NODE_CONTAINER_BIN_DIR)/confd:
 
 # Get the calico-bgp-daemon binary
 $(NODE_CONTAINER_BIN_DIR)/calico-bgp-daemon:
+	$(CURL) -L $(GOBGP_URL) -o $(@D)/gobgp
```

Comment:
why are we adding this? Don't think we need this guy here since we're using gRPC to get the status as opposed to GoBGP CLI. Doesn't seem like it's being used for the tests either, this would just make the image havier